osu.Game/Screens/SelectV2/BeatmapCarouselFilterGrouping.cs

```diff
@@ -102,6 +102,9 @@ public async Task<List<CarouselItem>> Run(IEnumerable<CarouselItem> items, Cance
                             lastGroupItem.NestedItemCount++;
                     }
 
+                    if (!BeatmapSetsGroupedTogether)
```

Comment:
Is there some meaning in this being directly below an `else` for the same condition?